zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sStandardInterpreterLauncher.java

```diff
@@ -70,6 +70,11 @@ boolean isRunningOnKubernetes() {
    * @throws IOException if namespace file could not be read
    */
   String getNamespace() throws IOException {
+    //namespace may be set from the interpreter properties
```

Comment:
The `K8sStandardInterpreterLauncher` is executed only on the Zeppelin server side.
Please rename this function to `getInterpreterNamespace()`.